lib/ransack/adapters/active_record/base.rb

```diff
@@ -23,9 +23,14 @@ def ransacker(name, opts = {}, &block)
         # Ransackable_attributes, by default, returns all column names
         # and any defined ransackers as an array of strings.
         # For overriding with a whitelist array of strings.
-        #
         def ransackable_attributes(auth_object = nil)
-          column_names + _ransackers.keys
+          ransackable_attributes = column_names + _ransackers.keys
+
+          if Ransack::SUPPORTS_ATTRIBUTE_ALIAS
+            ransackable_attributes += attribute_aliases.keys
+          end
+
+          ransackable_attributes
         end
```

Comment:
We could probably just inline the values here without memoizing `column_names + _ransackers.keys` in a local var:

```ruby
if Ransack::SUPPORTS_ATTRIBUTE_ALIAS
  column_names + _ransackers.keys + attribute_aliases.keys
else
  column_names + _ransackers.keys
end
```